app/src/components/panel/PanelToolBar.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  Bold,
+  Italic,
+  TextSize,
+  Quote,
+  Code,
+  Link,
+  List,
+} from '../../shared/icons'
+
+
+import { Button, TabBar, useTheme } from '@aragon/ui'
+
+const PanelToolBar = ({
+  screenIndex,
+  handleChange,
+  setSelectionBold,
+  setSelectionCode,
+  setSelectionItalic,
+  setSelectionLink,
+  setSelectionQuote,
+  setSelectionSize,
+  setSelectionUnorderedList,
+}) => {
+  const theme = useTheme()
+  return (
+    <EditToolBar>
+      <EditToolBarButton theme={theme} onClick={setSelectionSize} compact>
+        <TextSize />
+      </EditToolBarButton>
+      <EditToolBarButton theme={theme} onClick={setSelectionBold} compact>
+        <Bold />
+      </EditToolBarButton>
+      <EditToolBarButton theme={theme} onClick={setSelectionItalic} compact>
+        <Italic />
+      </EditToolBarButton>
+      <EditToolBarSeparator />
+      <EditToolBarButton theme={theme} onClick={setSelectionQuote} compact>
+        <Quote />
+      </EditToolBarButton>
+      <EditToolBarButton theme={theme} vonClick={setSelectionCode} compact>
```

Comment:
Typo here! Please use a linter before committing code to have some basic static analysis, these spots are hard to find without that tooling @PeterMPhillips 

It seems a priority to configure pre-commit lint hook to avoid missing these issues

```suggestion
      <EditToolBarButton theme={theme} onClick={setSelectionCode} compact>
```